### PR TITLE
feat: show message depending user package manager

### PR DESCRIPTION
**Pull request by brunocroh**
### Issue

#21425

### Short description

This PR implements, a better message when user, run the command
`playwright test --reporter=html `

Show a message with package manager of user instead of npx to everyone

if running with `yarn` output is 
```
To open last HTML report run:

yarn playwright show-report
```

if running with `npm` output is
```
To open last HTML report run:

npx playwright show-report
```

if running with `pnpm` output is
```
To open last HTML report run:

pnpm exec playwright show-report
```

### CODE REVIEW

**Comment by brunocroh** [comment]
@microsoft-github-policy-service agree

**Review by mxschmitt — CHANGES_REQUESTED** [review]
Please revert all the unrelated formatting changes.

**Comment by brunocroh** [comment]
@mxschmitt done, sorry for the mistake

File: packages/playwright-test/src/util.ts [hunk 1]
```diff
@@ -305,3 +305,12 @@ export function envWithoutExperimentalLoaderOptions(): NodeJS.ProcessEnv {
     result.NODE_OPTIONS = result.NODE_OPTIONS.replace(substring, '').trim() || undefined;
   return result;
 }
+
+export function determinePackageManager() {
+  if (process.env.npm_config_user_agent) {
+    if (process.env.npm_config_user_agent.includes('yarn')) return 'yarn';
```
Comment (pavelfeldman) [inline packages/playwright-test/src/util.ts:311]: We can use process.env.npm_config_user_agent? onstruct for brevity. Also I don't think `pnpm playwright` does the right thing, you probably want `pnpm exec playwright`.

Comment (brunocroh) [reply packages/playwright-test/src/util.ts]: I believe the right way to get package manage of use is this.
I changed the code to use `getFromENV` like another files of project
And change the message from `pnpm` to `pnpm exec`

If you have some suggest or believe this is not a better way to do that, let me know

**Comment by pavelfeldman** [comment]
Sorry we dropped the ball on this one and now it is obsolete.

**Pull request closed without merging**